resources/public/api/conf/3.0/self_assessment_charge_history_retrieve.yaml

```diff
@@ -22,9 +22,6 @@ get:
   parameters:
     - $ref: "./common/pathParameters.yaml#/components/parameters/nino"
     - $ref: "./common/pathParameters.yaml#/components/parameters/transactionId"
-      # {{#if (enabled 'chargeReferencePoaAdjustmentChanges')}}
-    - $ref: "./common/queryParameters.yaml#/components/parameters/chargeReference"
```

Comment:
I think chargeReference can also be deleted from queryParameters.yaml, looks like nothing else refers to it now.
When it's added back for the new endpoint, it'll be a path parameter.